### PR TITLE
Use Open Zeppelin test helpers. 

### PR DESCRIPTION
**Pull request by AndyWatt83**
The failures that we have seen running the tests on Travis were replicated on Matt's local machine, allowing easier testing to isolate the bug. It appears that the failing tests were the ones using the 'advanceMultipleBlocks' helper function.  Open Zeppelin have a number of very similar tests. It made sense to use them anyway, so I've switched over to their implementations. It'll be interesting to see if this passes on Travis..

### CODE REVIEW

File: test/TestCoinToss.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const CoinToss = artifacts.require('CoinToss');
-const helper = require('./helpers/delorean');
+const zephelpers = require('../node_modules/openzeppelin-solidity/test/helpers/advanceToBlock');
```
Comment (GarethOates) [inline test/TestCoinToss.js]: You don't need to use the direct path to reference a node module.  You can start with "openzeppelin-solidity" and it knows to look in the node_modules folder automatically.

Comment (AndyWatt83) [reply test/TestCoinToss.js]: Done, and it works now

**Comment by GarethOates** [comment]
@AndyWatt83 As I've made a change to the README, I need you to review this now.  I've reviewed your changes and accepted, so it's just for you to approve my readme change.

**Comment by AndyWatt83** [comment]
@GarethOates I don't think I can add a review to this, as I raised the PR. I can't add myself to the list of reviewers? Maybe you can add me?

**Pull request merged**